scenario/fixtures/docker-compose/.env

```diff
@@ -1,4 +1,7 @@
-DOCKER_IMG_TAG=:2.3
-PEER_IMG_TAG=:2.4
-FABRIC_CA_TAG=:1.4
-COUCHDB_IMG_TAG=:3.1.1
+# Environment variables will be overridden if set in user environment or Makefile.
+ORDER_IMAGE_TAG=2.3
+TOOLS_IMAGE_TAG=2.3
+PEER_IMAGE_TAG=2.4
+FABRIC_CA_IMAGE_TAG=1.5
+COUCHDB_IMAGE_TAG=3.1.1
+DOCKER_DEBUG=info:dockercontroller,gateway=debug
```

Comment:
I've re-added this without the problematic debug, so that people know how to set it and they can see the gateway debug once it is added.